tests/integration/local/start_api/test_start_api.py

```diff
@@ -683,4 +682,110 @@ def test_global_stage_variables(self):
 
         response_data = response.json()
 
-        self.assertEquals(response_data.get("stageVariables"), {"Stack": "Dev"})
+        self.assertEquals(response_data.get("stageVariables"), {'Stack': 'Dev'})
+
+
+class TestStartApiWithMethodsAndResources(StartApiIntegBaseClass):
```

Comment:
Should we have any tests with `{+proxy}` or whatever the way you define a proxy route? What about a route that has a path variable in it?